### ISSUE TITLE
update import <CookiesProvider>

### ISSUE THREAD

**Issue by lidforce** [moshest/next-client-cookies#18]
update the CookiesProvider import, the default import in VSCode is as follows: 
```TS
import { CookiesProvider } from "next-client-cookies";
```
and the import is considered deprecated. The readme says that you should specify 
`import { CookiesProvider } from "next-client-cookies/server";`

it took me a long time to figure out why the console was constantly displaying 
```
⨯ node_modules\next-client-cookies\dist\hook.js (44:0) @ eval
 ⨯ Error: Missing <CookiesProvider>
    at CreateUserPage (./src/app/admin/create-user/page.tsx:67:84)
null
```
Because of my inattention in the readme, I didn't understand what caused the error.

**Comment by moshest** [comment]
It's not clear what's the actionable item here. Can you explain?

The docs are up to date. Are you saying we need to update the error message?

**Comment by lidforce** [comment]
![изображение](https://github.com/moshest/next-client-cookies/assets/54450326/61ab7f7a-78de-47ac-a097-33f71d86e5bd)
I think it would be worth adding, when trying to import a component, to have it immediately imported from `next-client-cookies/server` instead of `next-client-cookies` as it is now.

**Comment by IgorCassolli** [comment]
![image](https://github.com/moshest/next-client-cookies/assets/37945750/e190fe50-7401-4aca-9f33-5f355f75f2cf)

Hi, someone can help me with this error?

**Comment by moshest** [comment]
@LidForce I think it will be solved on the next version, when we remove support for deprecated objects. I agree it would be better to just use a different name and avoid this confusion.

For now, I would update the error message to be:
```
Missing `<CookiesProvider>` from "next-client-cookies/server"
```

**Comment by moshest** [comment]
@IgorCassolli I'm afraid it's not relevant for this issue. Please check our `demo` folder for a working example and use StackOverflow for bugs on your setup. If you still think there is a bug on this library please open a new issue.

**Comment by moshest** [comment]
Released under `v1.1.1`
